securedrop_client/gui/widgets.py

```diff
@@ -1866,6 +1869,14 @@ def set_pending_styles(self):
         self.color_bar.setObjectName("ReplyWidget_status_bar_pending")
         self.color_bar.setStyleSheet(self.STATUS_BAR_CSS)
 
+    def _set_failed_to_decrypt_styles(self):
```

Comment:
It looks like the method to do this for message bubbles is called `set_error_styles` -- would it perhaps make sense to change it to `set_failed_to_decrypt_styles` (with or without `_`) as part of this PR, so that a developer looking to update both styles can more easily locate the corresponding method for message bubbles? (Also, it seems like the higher specificity is useful, even if there is just one error state for message bubbles for now.)

Reply:
yup, totally agree, and we should go further and rename "error" to "failed_to_decrypt"

Reply:
it's all about finding the right balance of bug-fixin and refactoring, and your suggestion is nice small refactor

Reply:
actually i have a purely-refactor-only follow-up pr that addresses your comment